diktat-rules/src/test/kotlin/org/cqfn/diktat/util/DiktatRuleSetProvider4Test.kt

```diff
@@ -61,9 +68,8 @@ class DiktatRuleSetProviderTest {
 
     companion object {
         private val ignoreFile = listOf(
-            "DiktatRuleSetProvider",
             "DiktatRule",
-            "IndentationError",
```

Comment:
Because I added a more generic way to filter out non-`Rule` classes (see above).

Otherwise, it becomes tedious to edit this ignore list each time I happen to add a new non-`Rule` class.

Don't worry, the integration test passes, just as before.